### PR TITLE
COMMERCE-11255 | Update APIs to store replacement product info

### PR DESCRIPTION
**Pull request by alessiorendina**
https://issues.liferay.com/browse/COMMERCE-11255

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-cart-content-web/src/main/java/com/liferay/commerce/cart/content/web/internal/portlet/action/AddCommerceOrderItemMVCActionCommand.java [hunk 1]
```diff
@@ -115,7 +115,7 @@ protected void doProcessAction(
 			CommerceOrderItem commerceOrderItem =
 				_commerceOrderItemService.addOrUpdateCommerceOrderItem(
 					commerceOrder.getCommerceOrderId(), cpInstanceId,
-					ddmFormValues, quantity, 0, commerceContext,
+					ddmFormValues, quantity, null, 0, commerceContext,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-cart-content-web/src/main/java/com/liferay/commerce/cart/content/web/internal/portlet/action/AddCommerceOrderItemMVCActionCommand.java:118]: why here is null?

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
QA Passed

**Comment by alessiorendina** [comment]
ci:forward

**Comment by pietrobocale** [comment]
QA Passed, after the conflicts are fixed, it's possible to forward

**Pull request closed without merging**